plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1694,6 +1694,7 @@ public void pause(final boolean v) throws IOException {
                                 LOGGER.log(Level.WARNING, null, t);
                             }
                         });
+                        cpsExec.getOwner().getListener().getLogger().close();
```

Comment:
We could also just `flush`, but I'm not really sure that it matters either way.
```suggestion
                        cpsExec.getOwner().getListener().getLogger().flush();
```